docs/LOCAL_DEPLOYMENT.md

```diff
@@ -30,6 +30,30 @@ The files for the dev container are located in `/.devcontainer/` folder.
 
 To customize the accelerator or run it locally, first, copy the `.env.sample` file to your development environment's `.env` file, and edit it according to [environment variable values table](#environment-variables) below.
 
+
+To run the accelerator in local you need to assign some roles to your principal id. You can do it either manually or programatically.
+
+### Manually assign roles
+You need to assign the following roles to your `PRINCIPALID` (you can get your 'principal id' from Microsoft Entra ID):
+1. Cognitive Services OpenAI Contributor role - a001fd3d-188f-4b5d-821b-7da978bf7442
+2. Search Service Contributor role -7ca78c08-252a-4471-8644-bb5ff32d4ba0
+3. Search Index Data Contributor - 8ebe5a00-799e-43f5-93ac-243d3dce84a7
+4. Storage Blob Data Reader role - 2a2b9908-6ea1-4ae2-8e65-a410df84e7d1 
+5. Reader role - acdd72a7-3385-48ef-bd42-f606fba81ae7
+
+### Programatically assign roles
+You can also update the `principalId` value with your own principalId in `deployment.bicep` file.
+Build the bicep file using the below command:
+
+```shell
+cd infra
+bicep build deployment.bicep
```

Comment:
Why do we need both steps - I thought `az deployment group create` would work with bicep files too

Reply:
Create the json file as a pre-commit hook #209